prow/config/jobs/operator.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
 
   - name: gencheck
     command: [make, gen-check]
-    modifiers: [optional]
+    modifiers: [optional, hidden]
```

Comment:
We're just about to fix this, so I'd rather not hide it. It's fine to hide it in 1.4 though.